jumanji/registration.py

```diff
@@ -61,10 +65,7 @@ def get_env_id(name: str, version: Optional[int] = None) -> str:
     Returns:
         The environment ID.
     """
-    version = version or 0
-    full_name = name + f"-v{version}"
-
-    return full_name
+    return name + f"-v{version}"
```

Comment:
I'm wondering. Should we check that `version` is an int here?

Reply:
The docstring says it must be an `int`. If the user doesn't respect the pre-condition then no guarantee. In any case, this function is only used internally and is not exposed to the user.